content/rust-lld-on-stable-1.90.0/index.md

```diff
@@ -0,0 +1,65 @@
++++
+path = "2025/08/29/rust-lld-on-1.90.0-stable"
+title = "Faster linking times with 1.90.0 stable on Linux using `rust-lld`"
+authors = ["Rémy Rakic"]
+
+[extra]
+team = "the compiler performance working group"
+team_url = "https://www.rust-lang.org/governance/teams/compiler#team-wg-compiler-performance"
++++
+
+TL;DR: rustc will use `rust-lld` by default on `x86_64-unknown-linux-gnu` on stable, starting in 1.90.0, to significantly reduce linking times. Test it out on beta now, and please report issues if you encounter any.
```

Comment:
```suggestion
TL;DR: rustc will start using the LLD linker by default on the `x86_64-unknown-linux-gnu` target starting with the Rust 1.90.0 stable release, which shoud significantly reduce linking times. Test it out on beta now, and please report any encountered issues.
```

Reply:
if you turn that "shoud" into "should" I'll commit the suggestion.

and then we can let the native english speakers correct us

Reply:
Thanks!